### ISSUE TITLE
syntax highlighting broken in feb release

### ISSUE THREAD

**Issue by jamesedwardsvtgov** [microsoft/vscode#177273]
Type: <b>Bug</b>

Reverting back to january restores syntax higlighting. 

I am noticing this in shell scripts with no file extension.

```
#!/usr/bin/env bash

temp="test"
echo "${temp}"
```

Jan:

![image](https://user-images.githubusercontent.com/102828994/225398378-ee9856aa-867b-44a9-8c2b-ae074f33896b.png)

Feb:

![image](https://user-images.githubusercontent.com/102828994/225397847-676bdc29-83e4-43a7-b980-8a82196b6194.png)

VS Code version: Code 1.76.2 (ee2b180d582a7f601fa6ecfdad8d9fd269ab1884, 2023-03-14T17:55:54.936Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: Yes

**Comment by RedCMD** [comment]
https://github.com/microsoft/vscode/issues/176190

**Comment by jamesedwardsvtgov** [comment]
@alexr00 thank you for this information! Will the improvement be available in the March 23 release?

**Comment by alexr00** [comment]
Yes, the latest shellscript grammar improvements will be in our March release.

**Comment by jeff-hykin** [comment]
@jamesedwardsvtgov You can also get the bleeding-edge updates of the shell syntax in any release by installing [Better Shell Syntax](https://marketplace.visualstudio.com/items?itemName=jeff-hykin.better-shellscript-syntax)
